spec/models/category_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
```

Comment:
- Hi, Please the test still failing. Kindly fix it as requested by the previous review. 
- 
![Capture](https://user-images.githubusercontent.com/63972670/204112869-6371bcf4-5e61-49dc-9249-facd30a9d1c2.JPG)